zone/spells.cpp

```diff
@@ -6770,3 +6776,11 @@ void Mob::SetHP(int64 hp)
 
 	current_hp = hp;
 }
+
+bool Mob::CheckWaterLoS(Mob* attacker, Mob* target) // checks if both attacker and target are both in or out of the water
```

Comment:
nit: Parameter target is triggering [clang-diagnostic-shadow] which means it is named the same as a property on Mob or its parent hierarchy. Consider renaming to something like spell_target or los_target.